apps/nextjs-website/src/lib/homepage.ts

```diff
@@ -79,17 +89,34 @@ export const makeHomepageProps = (
       ),
     },
   }),
-  productsShowcase: {
-    title: strapiHomepage.data.attributes.productsShowcase.title,
-    products: strapiHomepage.data.attributes.productsShowcase.products.data.map(
-      (product) => ({
-        name: product.attributes.name,
-        description: product.attributes.description,
-        slug: product.attributes.slug,
-        logo: product.attributes.logo.data.attributes,
-      })
-    ),
-  },
+  ...(strapiHomepage.data.attributes.ecosystem && {
+    ecosystem: {
+      title: strapiHomepage.data.attributes.ecosystem.title,
```

Comment:
The title can be undefined, so:
```suggestion
      title: strapiHomepage.data.attributes.ecosystem.title || '',
```